crates/stc_ts_type_checker/tests/conformance.pass.txt

```diff
@@ -1414,6 +1414,8 @@ expressions/binaryOperators/arithmeticOperator/arithmeticOperatorWithOnlyNullVal
 expressions/binaryOperators/arithmeticOperator/arithmeticOperatorWithTypeParameter.ts
 expressions/binaryOperators/arithmeticOperator/arithmeticOperatorWithUndefinedValueAndInvalidOperands.ts
 expressions/binaryOperators/arithmeticOperator/arithmeticOperatorWithUndefinedValueAndValidOperands.ts
+expressions/binaryOperators/comjsdoc/parseLinkTag.ts
```

Comment:
Then why did you modify this file?

Reply:
We know that location is invalid.

We suspect a bug was introduced while running the script

Reply:
Then please revert this file

Reply:
I get it.
Do you know anything about this anomaly?

Reply:
You should not commit `pass.txt` by its own.

Reply:
I was wondering why such a non-existent file would come out because I was committing the results of the script